README.md

```diff
@@ -23,8 +23,8 @@ Requirements for some features:
 - cuDNN support
   - cuDNN v2, v3
 - Caffe model support
-  - Python 2.7.6+ (Py3 is not supported)
   - Protocol Buffers (pip install protobuf)
+  - protobuf>=3.0.0 is required for Py3
```

Comment:
Line 27 is a supplemental comment about line 26. So line26 and line27 should not be listed in parallel. Rather, `pip install protobuf` and `probobuf>=3.0.0` should be in the same level.